Khan/perseus
# Update Polygon graphs to have weighted lines only after focus from keyboard

## Summary:
Previously, Polygon had its shape weighted upon hover and after clicking in the shape. This is not how circle is currently working. To bring them into alignment, this changes Polygon to become weighted only when the shape is tabbed to by the keyboard or if the shape is moved via keyboard after the user clicks on it. 

### Before
https://github.com/user-attachments/assets/16f3aef8-3312-4e0c-8815-542ad6ea7c98

### After
https://github.com/user-attachments/assets/b6058ae6-b31e-4942-ae24-9ae2853bfcc3

Issue: LEMS-2200

## Test plan:
- Checkout the associated branch
- Run `yarn dev` 
- On the gallery page, turn on the flag for polygon and circle
- Confirm Polygon does not have weighted lines on hover or when dragging. After clicking the shape, confirm the shape becomes weighted when you move it with the arrow keys. Confirm the lines return to normal after clicking outside of the shape.
- Confirm this functionality matches Circle graphs
